src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -208,9 +214,17 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
 
   @include pf-hidden-visible(var(--pf-c-toolbar__group--Display));
 
-  align-items: center;
+  align-items: var(--pf-c-toolbar__group--AlignSelf);
   margin-right: var(--pf-c-toolbar--spacer);
 
+  &.pf-m-align-self-center {
+    --pf-c-toolbar__group--AlignSelf: center;
```

Comment:
related to the other comment, the class is `align-self` but it's going to align its children. If the goal is to set group children alignment at the group level, I think we should use a `__group--AlignItems` var and `pf-m-align-items` modifier.